test/asyncio_tests/test_asyncio_client.py

```diff
@@ -67,6 +68,8 @@ async def test_close(self):
     async def test_unix_socket(self):
         if env.mongod_started_with_ssl:
             raise SkipTest("Server started with SSL")
+        if os.name == "nt":
+            raise SkipTest("Unix socket not available on Windows")
```

Comment:
Why this change? The windows tests are passing in EVG: https://evergreen.mongodb.com/waterfall/motor

Reply:
Removing the skip